.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Flutter CI
+
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+# on: push    # Default will running for every branch.
```

Comment:
why not for PRs as well ?

Reply:
What do you mean

Reply:
Running tests on branches, for pull requests ? To check they don't break anything before merging ?

Reply:
Updated 👍